### PR TITLE
Shorter title

### LINKED ISSUE #1292: Page title is too big for search results
Screenshot 2020-04-07 at 12 12 17 PM

Also says that is [not mobile friendly](https://search.google.com/test/mobile-friendly?url=https://excalidraw.com/&utm_source=gws&utm_medium=metaline&utm_campaign=notmobilefriendy)

### PR DESCRIPTION
**Pull request by lipis**
Fixes #1292

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -2,10 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <title>
-      Excalidraw | Virtual collaborative whiteboard for sketching hand-drawn
-      like diagrams
-    </title>
+    <title>Excalidraw | Hand-drawn look • Collaborative • Secure</title>
```
Comment (Fausto95) [inline public/index.html:5]:
```suggestion
    <title>Excalidraw | Hand-drawn looking diagrams • Collaborative • Secure</title>
```

Comment (lipis) [reply public/index.html:5]: I'm afraid it will be already long enough again.

File: public/index.html [hunk 2]
```diff
@@ -2,10 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <title>
-      Excalidraw | Virtual collaborative whiteboard for sketching hand-drawn
-      like diagrams
-    </title>
+    <title>Excalidraw | Hand-drawn looking diagrams • Collaborative • Secure</title>
```
Comment (lipis) [inline public/index.html:5]:
```suggestion
    <title>Excalidraw | Hand-drawn look & feel • Collaborative • Secure</title>
```

**Comment by dwelle** [comment]
The last update will not fit I think:

image

image

**Comment by lipis** [comment]
Let's see.. It's an easy fix. In the meantime, if @vjeux comes with better ideas...

**Pull request merged**